db/migrate/20180809155959_add_conversion_host_id_to_miq_tasks.rb

```diff
@@ -0,0 +1,6 @@
+class AddConversionHostIdToMiqTasks < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_tasks, :conversion_host_id, :bigint
```

Comment:
We wanted to add a method for throttling how many tasks run on a conversion host without having to loop through all active tasks and pull the conversion host out of the task options.

I don't see a trello card yet (/cc @fdupont-redhat) but there is an ongoing design doc here https://docs.google.com/document/d/1ANBPzDLF-id3MS9_8ztmrNjNwh3FJ6yGCvA2BUE808c/edit#heading=h.lzsppid3wjo9

Since this is still in design maybe we should hold off on this and come back and add this later.

Reply:
Here's the Trello card: https://trello.com/c/DOClRvfD/189-migration-throttling-cf-model